src/libraries/restore/analyzers/analyzers.depproj

```diff
@@ -5,6 +5,14 @@
     <Language>C#</Language>
   </PropertyGroup>
 
+  <ItemGroup>
+    <!-- 
+      Microsoft.CodeAnalysis.Common shouldn't be referenced directly but is needed by the analyzer depproj restore.
+      TODO: Remove when project restore is enabled for non test projects.
```

Comment:
Add link to the issue that tracks this work?

Reply:
done.